portfolio/src/main/webapp/contact.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
```

Comment:
Same comment as previous pull request. This should probably have a different title.

Reply:
Renamed!